frontend/packages/pipelines-plugin/src/components/repository/repository-utils.tsx

```diff
@@ -36,3 +36,7 @@ export const getGitProviderIcon = (url: string) => {
     }
   }
 };
+
+export const sanitizeBranchName = (branch: string): string => {
```

Comment:
maybe it should handled refs/heads code too like we do in the server code:

https://github.com/openshift-pipelines/pipelines-as-code/blame/a44ef08f70e12ea7d5bd5ef0b9ff816108038a07/pkg/formatting/vcs.go#L15-L23

Reply:
Added similar logic as server code.